packages/gatsby/src/utils/source-nodes.js

```diff
@@ -44,7 +44,7 @@ module.exports = async () => {
 
   // Garbage collect stale data nodes
   const touchedNodes = Object.keys(state.nodesTouched)
-  const staleNodes = _.values(state.nodes).filter(node => {
+  const staleNodes = Array.from(state.nodes.values()).filter(node => {
```

Comment:
I'm new here, but I believe that `_.values` is faster and less memory consuming than `Array.from` with `Map.values()`, was it compared?